flask_rebar/rebar.py

```diff
@@ -464,6 +467,8 @@ def add_handler(
             Arbitrary strings to tag the handler with. These will translate to Swagger operation tags.
         :param Type[USE_DEFAULT]|None|str mimetype:
             Content-Type header to add to the response schema
+        :param bool hidden:
+            Used to control the generation of swagger documentation of APIs
```

Comment:
I would update this comment to indicate how behavior changed by this parameter. "if hidden, documentation is not created for this request handler by default" or similar